src/ui/public/notify/notifier.js

```diff
@@ -314,6 +315,27 @@ Notifier.prototype.info = function (msg, cb) {
   }, cb);
 };
 
+/**
+ * Display a banner message
+ * @param  {String} msg [description]
+ * @return {[type]}     [description]
```

Comment:
Can you add the return type and descriptions in this docblock (my preference) or remove the generic defaults?